Assets/HandDrower.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class HandDrower : MonoBehaviour
+{   
+    public readonly static int handLimit = 4;
+
+    public GameObject simpleCardPrefab;
+    public GameObject[] facesPrefabs;
+
+    public static Vector3 calculateStartHandPosition(Vector3 handPositionCoordinate){
+        int left_side = HandDrower.handLimit / 2;
+        return new Vector3(x: handPositionCoordinate.x - left_side * 1.5f, y: handPositionCoordinate.y, z: handPositionCoordinate.z);
```

Comment:
```suggestion
        return handPositionCoordinate - left_side * 1.5f * Vector3.left;
```

Reply:
It's wrong: elft vector means (-1, 0, 0). So in that case i'll move card to the right instead of left.

Reply:
Good catch tho, fixed